ieeeuoft/makeuoft-2023
# Improved order fulfillment on admin side for hss

## Overview

- allows admins to only fulfill some order items instead of all
- changed some styling on admin team detail page


## Unit Tests Created

- none

## Steps to QA

- create an order as a participant
- fulfill only part of that order on the HSS
- check to see if that is reflected on the participant side and in the inventory, items will only be deducted from the inventory when orders are picked up (e.g. if you fulfill part of an order and put it in Ready for Pickup, the full quantity will still be deducted from inventory, until the order is set to Picked Up)
- test for other edge cases
- test changing order for all statuses
